### ISSUE TITLE
Multiple Accounts or Multiple Instances on same OS Session

### ISSUE THREAD

**Issue by fedenelli** [alexkim205/G-Desktop-Suite#41]
I was wondering if there is any way to run multiple accounts of G Suite with this project.
I have one personal account and two business accounts, and I have been struggling with solutions like Franz, Station and Wavebox.

So, is there a way to install 3 different instances of G Desktop Suite in the same MacOS session?

Thank you so much for this awesome project.

**Comment by alexkim205** [comment]
This is a interesting use case I hadn't thought about too much. Google already gives you a way to switch between accounts that you're already signed in. Would you want to have multiple accounts running at the same time?

**Comment by fedenelli** [comment]
Exactly! Much of the time I need to have two of my inboxes in the screen at the same time, and receive notifications for all my accounts in order to answer almost immediately.

I know this is not a feature for the average user, but I was wondering if there is a workaround that doesn't require too much code to make it work (such as multiple standalone installs)

**Comment by nathang21** [comment]
I do I.T. consulting and have about 10+ chrome profiles (each with 1 G Suite account) for all my different clients. Having to re-log into each one in this app will be annoying, but once done would be very useful to be able to switch accounts easily, or have multiple logged in simultaneously. 

Otherwise i'd only be able to use this with my personal account, and having a separate workflow would be less useful.
